tests/test_codex_feed_hooks.py

```
@@ -78,6 +103,289 @@ def _run(self) -> None:
                     conn.sendall(json.dumps(response).encode("utf-8") + b"\n")
 
 
+def monitor_pids_for_session(session_id: str) -> list[int]:
+    result = subprocess.run(
+        ["ps", "-axo", "pid=,command="],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)